### ISSUE TITLE
totalPages is off by 1

### ISSUE THREAD

**Issue by laceysanderson** [UofS-Pulse-Binfo/tripal_ws_brapi#38]
The number of pages is off by one. **NOTE: The totalPages is the number of pages not the index of the last page. As such the fact that BrAPI starts pages off at 0 does not impact the total pages.**

```
      // NOTE: BrAPI page count starts at 0.
      // Account for when page is one and only, it should not minus 1.
      $total_page = ($total_page > 1) ? $total_page - 1 : 1;
```
https://github.com/UofS-Pulse-Binfo/tripal_ws_brapi/blob/master/class/TripalWebServiceCall.inc#L733-L735

Example)
Page: http://localhost/web-services/brapi/v1/breedingmethods?pageSize=1
There are a total of 4 results. This with only one result per page there will be 4 pages. However, the response says 3.
<img width="778" alt="Screen Shot 2021-01-03 at 5 25 54 PM" src="https://user-images.githubusercontent.com/1566301/103491345-c0f58780-4de8-11eb-8adc-2fe340865f5c.png">

**Comment by laceysanderson** [comment]
NOTE: Once fixed, uncomment totalPage size assertion in breeding call tests.

**Comment by laceysanderson** [comment]
Fixed by PR #40
